src/org/folio/models/EurekaTenant.groovy

```diff
@@ -0,0 +1,52 @@
+package org.folio.models
+
+/**
+ * EurekaTenant class representing a tenant configuration for Eureka.
+ * It provides chainable setter methods following builder pattern for ease of use.
+ */
+class EurekaTenant extends Tenant {
+  /**
+   * Keycloak client identifier.
+   */
+  String clientId
+
+  /**
+   * Some secret phrase to authenticate in Keycloak.
+   * Is stored in AWS SSM Parameter Store or Hashicorp Vault.
+   */
+  String clientSecret
+
+  /**
+   * Keycloak service URL.
+   * Is the same for all Tenants
+   */
+  String keycloakUrl
+
+  /**
+   * Kong service URL.
+   * Is the same for all Tenants
+   */
+  String kongUrl
+
+  /**
+   * Tenant Manager Service URL.
+   * Is the same for all Tenants.
+   */
+  static String tenantManagerUrl = "http://mgr-tenants"
```

Comment:
Why we need such url? I this will be used to sending request? If yes, then it is no sence as from jekins we will not be able to access such url, as it namespace-scope
I think we need just Kong and Keycloak urls which will be exposed

Reply:
@eldiiar-duishenaliev 
do you know if **http://mgr-tenants** service is accessible via **Kong** somehow?

Reply:
as an example please review this link: https://[folio-eureka-scout-kong.ci.folio.org/tenants](https://folio-eureka-scout-kong.ci.folio.org/tenants)

Reply:
leave it without default value please.

Reply:
Code is adjusted accordingly to your recommendations.